src/main/java/edu/ucsb/cs56/mapache_search/VoteController.java

```diff
@@ -0,0 +1,26 @@
+package edu.ucsb.cs56.mapache_search;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.oauth2.client.authentication.OAuth2AuthenticationToken;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+
+import edu.ucsb.cs56.mapache_search.repositories.VoteRepository;
+
+
+@Controller
+public class VoteController {
```

Comment:
This is currently an empty class. We may want to move the /updateVote endpoint to this class in future.